### PR TITLE
Metrics

### PR DESCRIPTION
**Pull request by s0nicboOm**
Explain what this PR does.

### CODE REVIEW

File: tests/monitoring/__pycache__/test_metrics.cpython-311-pytest-7.4.4.pyc
Comment (vigith) [inline tests/monitoring/__pycache__/test_metrics.cpython-311-pytest-7.4.4.pyc:1]: remove this and add `__pycache_` to gitignore?

File: .idea/.gitignore
Comment (vigith) [inline .idea/.gitignore:1]: wrong place

**Review by vigith — COMMENTED** [review]
what does PR do? Only metrics parsing? How will the inference and training be done?

**Comment by s0nicboOm** [comment]
Yes. I am still in process of cleaning this. But all inference and training now happens through numalogic codebase. This repo will be using ML prom metrics for numalogic codebase.

File: tests/monitoring/__pycache__/test_metrics.cpython-311-pytest-7.4.4.pyc
Comment (s0nicboOm) [reply tests/monitoring/__pycache__/test_metrics.cpython-311-pytest-7.4.4.pyc:1]: Yes. I am still in process of cleaning this. But all inference and training now happens through numalogic codebase. This repo will be using ML prom metrics for numalogic codebase.


Have fixed these changes.

**Review by ab93 — COMMENTED** [review]
Where are all the workflows?

**Comment by s0nicboOm** [comment]
> Where are all the workflows?

Have added them. Codecov I think I will need your help to access settings and add the codecov token there. Otherwose all workflows are working good.

**Comment by s0nicboOm** [comment]
not needed

**Pull request closed without merging**